renderlib/graphics/gl/Image3D.cpp

```diff
@@ -253,9 +253,9 @@ Image3D::prepareTexture(Scene& s)
   // glGenTextures(1, &_textureid);
   glBindTexture(GL_TEXTURE_3D, m_textureid);
   check_gl("Bind texture");
-  glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_MIN_FILTER, GL_LINEAR);
+  glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_MIN_FILTER, useLinearInterpolation ? GL_LINEAR : GL_NEAREST);
   check_gl("Set texture min filter");
-  glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_MAG_FILTER, GL_LINEAR);
+  glTexParameteri(GL_TEXTURE_3D, GL_TEXTURE_MAG_FILTER, useLinearInterpolation ? GL_LINEAR : GL_NEAREST);
```

Comment:
these changes are the implementation for the "ray march" renderer, same as what the path trace renderer has to do